build/dist

```diff
@@ -296,7 +307,9 @@ done
 cp "$KYUUBI_HOME/externals/kyuubi-flink-sql-engine/target/kyuubi-flink-sql-engine_${SCALA_VERSION}-${VERSION}.jar" "$DISTDIR/externals/engines/flink/"
 
 # Copy spark engines
-cp "$KYUUBI_HOME/externals/kyuubi-spark-sql-engine/target/kyuubi-spark-sql-engine_${SCALA_VERSION}-${VERSION}.jar" "$DISTDIR/externals/engines/spark/"
+for SUPPORT_VERSION in 2.12 2.13; do
```

Comment:
SUPPORT_VERSION is not a good var name here, as SCALA_VERSION is occupied, simplify use `i`